device_coupler/config/device_coupler_config.yaml

```diff
@@ -0,0 +1,6 @@
+trunk_iface: trunk0
```

Comment:
as discussed, I think some of these are redundant with existing parameters (test_vlans, grpc_port), and then I'd find the right way to fold the others into the existing setup. E.g. the "daq_grpc_ip" argument really should be collocated with the (existing) server_port parameter (e.g. server_ip).

Reply:
Changed device coupler so that it reads. DAQ. config instead